frontend/src/modules/organization/pages/organization-merge-suggestions-page.vue

```diff
@@ -0,0 +1,270 @@
+            class="btn btn--primary btn--lg w-full"
+            :loading="sendingMerge"
+            @click="mergeSuggestion()"
+          >
+            Merge organizations
+          </el-button>
+        </div>
+      </div>
+    </div>
+    <!-- Empty state -->
+    <div v-else class="pt-20 flex flex-col items-center">
+      <div
+        class="ri-shuffle-line text-gray-200 text-10xl h-40 flex items-center mb-8"
+      />
+      <h5 class="text-center text-lg font-semibold mb-4">
+        No merge suggestions
+      </h5>
+      <p class="text-sm text-center text-gray-600 leading-5">
+        We couldn’t find any duplicated organizations
+      </p>
+    </div>
+  </app-page-wrapper>
+</template>
+
+<script setup>
+import { ref, onMounted, computed } from 'vue';
+import Message from '@/shared/message/message';
+import { mapGetters } from '@/shared/vuex/vuex.helpers';
+import AppLoading from '@/shared/loading/loading-placeholder.vue';
+import AppOrganizationMergeSuggestionsDetails from '@/modules/organization/components/suggestions/organization-merge-suggestions-details.vue';
+import { OrganizationService } from '../organization-service';
+import { OrganizationPermissions } from '../organization-permissions';
+
+const { currentTenant, currentUser } = mapGetters('auth');
+
+const organizationsToMerge = ref([]);
+const primary = ref(0);
+const offset = ref(0);
+const count = ref(0);
+const loading = ref(false);
+
+const sendingIgnore = ref(false);
+const sendingMerge = ref(false);
+
+const bioHeight = ref(0);
+
+const isEditLockedForSampleData = computed(
+  () => new OrganizationPermissions(currentTenant.value, currentUser.value)
+    .editLockedForSampleData,
+);
+
+const confidence = computed(() => {
+  if (organizationsToMerge.value.similarity >= 0.8) {
+    return {
+      color: '#059669',
+      svg: `<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.1667 2.66699H17.5V17.5003H14.1667V2.66699ZM8.33337 7.5H11.6667V17.5003H8.33337V7.5Z" fill="#059669"/>
+<path d="M2.5 12H5.83333V17.5H2.5V12Z" fill="#059669"/>
+</svg>`,
+    };
+  }
+  if (organizationsToMerge.value.similarity >= 0.6) {
+    return {
+      color: '#3B82F6',
+      svg: `<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.1666 2.66699H17.4999V17.5003H14.1666V2.66699ZM8.33325 7.5H11.6666V17.5003H8.33325V7.5Z" fill="#D1D5DB"/>
+<path d="M8.33325 7.5H11.6666V17.5003H8.33325V7.5Z" fill="#3B82F6"/>
+<path d="M2.5 12H5.83333V17.5H2.5V12Z" fill="#3B82F6"/>
+</svg>`,
+    };
+  }
+  return {
+    color: '#D97706',
+    svg: `<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.1667 2.66699H17.5V17.5003H14.1667V2.66699ZM8.33337 7.5H11.6667V17.5003H8.33337V7.5Z" fill="#D1D5DB"/>
+<path d="M2.5 12H5.83333V17.5H2.5V12Z" fill="#F59E0B"/>
+</svg>`,
+  };
+});
```

Comment:
These is exactly the same as members, should we add this svg to the `crowd-icons` file or maybe add it to an external component?